packages/app/src/composables/usePreferences.ts

```diff
@@ -9,12 +9,17 @@ mutation Preferences_SetPreferences ($value: String!) {
   setPreferences (value: $value)
 }`
 
+type Preference = 'autoScrollingEnabled' | 'isSpecsListOpen' | 'reporterWidth' | 'specsListWidth'
```

Comment:
No need to repeat the list of preferences, TypeScript is good at that.

```suggestion
type Preference = keyof RunnerUiStore
```

Reply:
nice